bigquery_etl/docs/derived_datasets/templates/table.md

```diff
@@ -1,10 +1,8 @@
-## [{{ table_name }}](#{{ table_name }})
+## {{ title }}
```

Comment:
Gotcha thanks for the context. I think we'll have to generate the link dynamically since some of the titles are table friendly names (for readability). For eg. the title: `Events unnested view for Mozilla VPN` should link to to `#events-unnested-view-for-mozilla-vpn`. Will make that update

Reply:
Okay it turns out slugs are hard (non-unique friendly names, not sure how to get them from mkdocs directly) so I reverted that part and just bolded the friendly name. Good catch!